@navikt/core/react/src/form/combobox/FilteredOptions/filteredOptionsContext.tsx

```diff
@@ -76,7 +73,7 @@ export const FilteredOptionsProvider = ({
   const [isInternalListOpen, setInternalListOpen] = useState(false);
   const { customOptions } = useCustomOptionsContext();
 
-  const filteredOptions = useMemo(() => {
+  const filteredOptions = useMemo<ComboboxOption[]>(() => {
```

Comment:
Tror ikke det er nødvendig å definere typen eksplisitt her
```suggestion
  const filteredOptions = useMemo(() => {
```

Reply:
Hmm, det var det på et tidspunkt, men det var sikkert underveis i en refaktorering.